src/server/game/Entities/Player/Player.cpp

```diff
@@ -22414,7 +22414,7 @@ void Player::ApplySpellMod(uint32 spellId, SpellModOp op, T& basevalue, Spell* s
         return;
 
     float totalmul = 1.0f;
-    int32 totalflat = 0;
+    T totalflat = 0;
```

Comment:
Why change this?

Reply:
I thought it was a mistake.
Isn't it? shouldn't it be T?